pkg/dao/snapshots_test.go

```diff
@@ -215,7 +289,7 @@ func (s *SnapshotsSuite) TestListNotFound() {
 
 	s.createSnapshot(rConfig)
 
-	collection, total, err := sDao.List("bad-uuid", pageData, filterData)
+	collection, total, err := sDao.List(rConfig.OrgID, "bad-uuid", pageData, filterData)
```

Comment:
Could you add an additional test to "TestListNotFound" that creates has a bad org_id, but has a good repository uuid? This way we have to test to check that the query is filtered by org_id.

Unless you have a test elsewhere for that and I missed it.

Reply:
Added TestListNotFoundBadOrgId! 
Thanks for the call out! 🍻